modules/ROOT/pages/debugging-kernel-crashes.adoc

```diff
@@ -0,0 +1,40 @@
+= Enabling kdump to debug kernel crashes
+
+. If `kexec-tools` is not present on the system, install `kexec-tools` using `rpm-ostree`, and reboot as needed.
+[source, bash]
++
+----
+sudo rpm-ostree install kexec-tools
+sudo systemctl reboot
+----
```

Comment:
I think we can drop this now that we're planning to ship `kexec-tools` in-tree.

Reply:
What about for users that didn't update? But I guess everyone knows how to do an `rpm-ostree install` :)

Reply:
I think in general the docs assume that the user is at least at the latest stable release. But yeah, on that note we should hold this PR until https://github.com/coreos/fedora-coreos-config/pull/708 actually makes it to the stable stream.